contents/docs/libraries/js/index.mdx

```diff
@@ -342,6 +342,74 @@ if (posthog.isFeatureEnabled('new-groups-feature')) {
 
 To check flag status for a different group, first switch the active group by calling `posthog.group()`.
 
+## Surveys
+
+[Surveys](/docs/surveys) launched with [popover presentation](/docs/surveys/creating-surveys#display-mode) are automatically shown to users matching the targeting you set up.
+
+Surveys using the **API** presentation enable you to implement your own survey UI and use PostHog to handle display logic, capturing results, and analytics.
+
+This starts by fetching active surveys using the following methods with a callback function:
+
+```js-web
+// Fetch enabled surveys for the current user
+posthog.getActiveMatchingSurveys(callback, forceReload) 
+
+// Fetch all surveys
+posthog.getSurveys(callback, forceReload)
+```
+
+The response is cached by default. To force a reload, pass `true` as the `forceReload` argument.
+
+The response an array of survey objects that look like this:
+
+```json
+[{
+  "id": "your_survey_id",
+  "name": "Your survey name",
+  "description": "Metadata describing your survey",
+  "type": "api", // can be either "api" or "popover"
```

Comment:
or feedback? Not sure, havent seen the implementation of it

Reply:
Just checked, it will be `"widget"`